src/pyoauth/oauth/base.py

```diff
@@ -0,0 +1,54 @@
+import json
+import urllib
+from urllib.parse import urlencode
+from urllib.request import Request, urlopen
```

Comment:
Check if this can be done like
`from urllib import parse, request`. So, reducing two imports to one import.